docs/features/record-routing.mdx

```diff
@@ -0,0 +1,177 @@
+  one source and one destination connector.
+* Route data from one collection (table, topic, index, etc.) to multiple
+  collections on a single server.
+* Route data to different instances or different systems altogether.
+
+This can be done it two ways:
+
+* by using a connector that supports writing to multiple collections
+* by using filter processors
+
+## Using a connector that supports writing to multiple collections
+
+The [OpenCDC format](/docs/features/opencdc-record) recommends that a record's
+source or destination collection is written to its metadata using
+the [`opencdc.collection` key](/docs/features/opencdc-record#opencdccollection).
+
+A destination connector can then utilize that information and write the record
+to the correct collection.
+
+:::note
+It's the connectors that eventually provide the ability to read from multiple
+source collections, or write to multiple destination collections. Consult the
+connector documentation to check if it supports this feature.
+:::
+
+### Example: Route data from multiple Kafka topics to multiple PostgreSQL tables on a single database server
+
+Let's assume we have a pipeline that streams data from two Kafka
+topics, `employees_finance` and `employees_legal` to PostgreSQL (both of the
+connectors support reads from multiple collections and writes to multiple
+collections). A pipeline configuration file could look like this:
+
+```yaml
+---
+version: 2.2
+pipelines:
+  - id: example-pipeline
+    status: running
+    name: example-pipeline
+    description: 'Kafka to PostgreSQL'
+    connectors:
+      - id: kafka-source
+        type: source
+        plugin: "builtin:kafka"
+        name: kafka-source
+        settings:
+          servers: "localhost:9092"
+          topics: "employees_finance,employees_legal"
+
+      - id: pg-destination
+        type: destination
+        plugin: "builtin:postgres"
+        name: pg-destination
+        settings:
+          url: "postgresql://username:password@localhost/testdb?sslmode=disable"
+          table: "employees"
+```
+
+The Kafka connector will read records like the ones below:
+
+```json lines
+{
+    "metadata": {
+        "opencdc.collection": "employees_finance",
+        // rest of metadata
+    },
+    "payload": {
+        "after": {
+            "name": "John"
+        }
+    }
+    // other record fields
+}
+{
+    "metadata": {
+        "opencdc.collection": "employees_legal",
+        // rest of metadata
+    },
+    "payload": {
+        "after": {
+            "name": "Adam"
+        }
+    }
+    // other record fields
+}
+```
+
+The PostgreSQL destination connector will check the value in
+the `opencdc.collection` metadata field. Based on that, it will write John's
+information to the `employees_finance` table and Adam's information to
+the `employees_legal` table.
+
+If the `opencdc.collection` metadata field is missing, the record will then be
+written to the default table, which is `employees` in our case.
```

Comment:
This is not correct. This was the old behavior, where the connector looked at the metadata field `postgres.table` and used the connector configuration parameter `table` as a fallback, if that metadata field was not found.

Now we leave it to the user to configure how they want the connector to figure out the target table. The configuration parameter `table` accepts a Go template, which will be evaluated for each record, the resulting string will be used as the target table. By default, the configuration parameter is set to `{{ index .Metadata "opencdc.collection" }}`, meaning that by default we try to route the record based on the metadata field `opencdc.collection`. However, if you set `table` to `employees`, then all records will be written into `employees`, including ones that contain the aforementioned metadata field.

Reply:
Ah, I see, so no fallback.